src/constants/wallets.ts

```
@@ -5,14 +5,12 @@ import { type onboarding } from 'starboard-client-js';
 import { STRING_KEYS } from '@/constants/localization';
 
 import {
+  BakoSafeIcon,
   CoinbaseIcon,
```

ESLint

[error] 'CoinbaseIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)